src/app/core/submission/submission-rest.service.ts

```diff
@@ -65,9 +66,15 @@ export class SubmissionRestService {
    *    The base endpoint for the type of object
    * @param resourceID
    *    The identifier for the object
+   * @param collectionId
+   *    The owning collection for the object
    */
-  protected getEndpointByIDHref(endpoint, resourceID): string {
-    return isNotEmpty(resourceID) ? `${endpoint}/${resourceID}` : `${endpoint}`;
+  protected getEndpointByIDHref(endpoint, resourceID, collectionId?: string): string {
+    let url = isNotEmpty(resourceID) ? `${endpoint}/${resourceID}` : `${endpoint}`;
+    if (collectionId) {
+      url = new URLCombiner(url, '?owningCollection=' + collectionId).toString();
```

Comment:
Not really an issue, just an FYI: in typescript (and post-2015 javascript) [string templating ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)is built in. You can write the same thing like this 
```suggestion
      url = new URLCombiner(url, `?owningCollection=${collectionId}`).toString();
```